Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -372,14 +400,17 @@ - (void)_installScreenTimeWebpageController
         [_screenTimeWebpageController setSuppressUsageRecording:![_configuration websiteDataStore].isPersistent];
 
         RetainPtr screenTimeView = [_screenTimeWebpageController view];
```

Comment:
I would leave a comment about why this is created, regardless of `[_configuration _showsSystemScreenTimeBlockingView]`.